services/core/shared/core/queue.js

```diff
@@ -0,0 +1,61 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2019 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+const { Logger } = require('lisk-service-framework');
+const Queue = require('bull');
+const util = require('util');
+
+const config = require('../../config');
+
+const logger = Logger();
+
+const initializeQueue = (queueName, queueProcess) => {
+	const statsQueue = new Queue(queueName, {
+		redis: config.endpoints.redis,
+		limiter: {
+			max: 8,
+			duration: 20,
+		},
+		prefix: 'Queue',
```

Comment:
The queue prefix could be taken from `package.json -> name`.
Like `queue-${packageJsonName}` or so.

Reply:
Done.